org.gimp.GIMP.json

```diff
@@ -16,12 +16,34 @@
                     "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
                     "--talk-name=org.freedesktop.FileManager1"],
     "tags": ["stable"],
-    "cleanup": ["/include", "/lib/pkgconfig", "/lib/cmake/", "/share/pkgconfig",
-                "/share/aclocal", "/man", "/share/man", "/share/gtk-doc", "/share/doc",
-                "/share/vala", "*.la", "*.a", "/bin/wmf*", "/bin/libwmf-*",
-                "/share/pygobject/2.0/codegen",
-                "/bin/pygtk*", "/bin/pygobject*", "/lib/pygtk",
-                "/bin/python*-config", "/bin/pip*", "/bin/smtpd.py" ],
+    "cleanup": [
+        "/lib/cmake/",
+        "/man", "/share/man",
+        "/share/gtk-doc",
+        "/share/doc",
+        "/share/vala",
+        "*.la",
+        "*.a",
+        "/bin/wmf*",
+        "/bin/libwmf-*",
+        "/share/pygobject/2.0/codegen",
+        "/bin/pygtk*",
+        "/bin/pygobject*",
+        "/lib/pygtk",
+        "/bin/python*-config",
+        "/bin/pip*",
+        "/bin/smtpd.py"
+    ],
+    "add-extensions": {
+        "org.gimp.GIMP.Plugin": {
+            "version": "2-3.36",
+            "directory": "extensions",
+            "add-ld-path": "lib",
+            "merge-dirs": "plug-ins",
```

Comment:
See above for the content of `/app/extensions/plug-ins/`

And that lead to a question I forgot and something that needs to be done: Setting the default path for that in GIMP.

Reply:
> And that lead to a question I forgot and something that needs to be done: Setting the default path for that in GIMP.

It could be added to the list of default paths, but rather than doing this, would `"merge-dirs"` allow to merge the extension `plug-ins/` with `/app/lib/gimp/2.0/plug-ins/`? This may be better, in the logics that plug-ins installed as flatpak extensions can also be considered system plug-ins (not modifyable, etc.).

Reply:
The problem is that it is already populated so I'm not sure if that's possible to use `/app/lib/gimp/2.0` as the extension point. I didn't try though.